packages/merchant-server/src/l2/wallet.ts

```diff
@@ -0,0 +1,27 @@
+// @ts-ignore
+import {ZkTx, ZkAddress} from "@zkopru/transaction";
+// @ts-ignore
+import {F} from '@zkopru/babyjubjub'
```

Comment:
I believe that you are going to make this without this typescript comment.
As far as I know `@zkopru/babyjubjub` cannot be imported without this comment, or without manually copying declaration files like this script.
https://github.com/zkopru-network/stress-test/blob/main/scripts/copy-files.js

Reply:
FYI, the reason is that typescript does not copy declaration files under the source directory when compiling. just generated `.d.ts` from `ts`.
the declaration files in `zkopru/packages/babyjubjub/src` and `zkopru/packages/contracts/src` are pre-generated, which are made by typechain, in other words, that made by contract compile.

Reply:
thank you! it already been solved. now I am using npm distribution.